lib/grape/middleware/formatter.rb

```diff
@@ -110,7 +110,7 @@ def read_rack_input(body)
           rescue Grape::Exceptions::Base => e
             raise e
           rescue StandardError => e
-            throw :error, status: 400, message: e.message
+            throw :error, status: 400, message: e.message, backtrace: e
```

Comment:
Is this a standard pattern somehow in Ruby? I mean I would expect `backtrace` to be an actual backtrace and not the original exception. Is anyone doing it like this elsewhere?

Are there other places we `throw :error` like this that also need fixing?